executor/syscalls_akaros.h

```diff
@@ -8,41 +9,41 @@
 #define SYZ_DATA_OFFSET 536870912
 unsigned syscall_count = 35;
 call_t syscalls[] = {
-    {"abort_sysc_fd", 33},
-    {"chdir", 116},
-    {"close", 103},
-    {"fchdir", 124},
-    {"fcntl$F_DUPFD", 107},
-    {"fcntl$F_DUPFD_CLOEXEC", 107},
-    {"fcntl$F_GETFD", 107},
-    {"fcntl$F_GETFL", 107},
-    {"fcntl$F_GETLK", 107},
-    {"fcntl$F_GETOWN", 107},
-    {"fcntl$F_SETFD", 107},
-    {"fcntl$F_SETFL", 107},
-    {"fcntl$F_SETLK", 107},
-    {"fcntl$F_SETLKW", 107},
-    {"fcntl$F_SETOWN", 107},
-    {"fstat", 104},
-    {"getcwd", 117},
-    {"link", 112},
-    {"llseek", 111},
-    {"lstat", 106},
-    {"mkdir", 118},
-    {"mmap", 18},
-    {"mprotect", 20},
-    {"munmap", 19},
-    {"nanosleep", 36},
-    {"openat", 102},
-    {"read", 100},
-    {"readlink", 115},
-    {"rename", 123},
-    {"rmdir", 119},
-    {"stat", 105},
-    {"symlink", 114},
-    {"unlink", 113},
-    {"waitpid", 17},
-    {"write", 101},
+	{"abort_sysc_fd", 33},
```

Comment:
The docs change looks good to me now, but please remove these unrelated chances from the patch.

Reply:
These were the changes due to "make install pre_requisites".
Please tell me what I should do about it.

Reply:
Something like:
```
git checkout HEAD^ executor/syscalls_akaros.h && git commit --amend && git push -f
```
should help.

Reply:
This is altering the master branch. I need to first fix my fork then do it right? I dont exactly know the command for that..

Reply:
I am denied the permission to do so. What else can I do?